R/parameters.R

```diff
@@ -0,0 +1,210 @@
+#' Read in disease process parameters from an external file or files
+#'
+#' @param generation_interval_path,delay_interval_path,right_truncation_path
+#'   Path to a local file with the parameter PMF. See [read_interval_pmf] for
+#'   details on the file schema. The parameters can be in the same file or a
+#'   different file.
+#' @param disease One of `COVID-19` or `Influenza`
+#' @param as_of_date Use the parameters that were used in production on this
+#'   date. Set for the current date for the most up-to-to date version of the
+#'   parameters and set to an earlier date to use parameters from an earlier
+#'   time period.
+#' @param state Used only for parameters with a state-level estimate (i.e., only
```

Comment:
Rephrase to be inclusive of territories?

Reply:
This may be too much future-proofing. We're not actually producing estimates for the territories currently, but we may in the future.

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/16/commits/e1492bd89ec2115333fbf86b58911d6d80ee7a36